docs/handbook/01_intro.md

```diff
@@ -0,0 +1,19 @@
+
+The _Munich Quantum Toolkit (MQT)_, which is developed by the [Chair for Design Automation](https://www.cda.cit.tum.de/) at the [Technical University of Munich](https://www.tum.de/), aims to leverage this latent potential by providing a collection of state-of-the-art design automation methods and software tools for quantum computing.
+Our overarching objective is to provide solutions for design tasks across the entire quantum software stack.
+This entails high-level support for end users in realizing their _applications_ {cite:p}`quetschlichAutomatedFrameworkRealizing2023,quetschlichHybridClassicalQuantum2023,quetschlichPredictingGoodQuantum2023,quetschlichUtilizingResourceEstimation2024,quetschlichApplicationawareQuantumCircuit2024, quetschlichMQTPredictorAutomatic2023` as well as efficient methods for the _classical simulation_ {cite:p}`zulehnerAdvancedSimulationQuantum2019,zulehnerMatrixVectorVsMatrixMatrix2019,hillmichJustRealThing2020,hillmichAccurateNeededEfficient2020,hillmichApproximatingDecisionDiagrams2022,grurlConsideringDecoherenceErrors2020,grurlStochasticQuantumCircuit2021,grurlNoiseawareQuantumCircuit2023,burgholzerHybridSchrodingerFeynmanSimulation2021,sanderHamiltonianSimulationDecision2023,matoMixeddimensionalQuantumCircuit2023,hillmichConcurrencyDDbasedQuantum2020,burgholzerExploitingArbitraryPaths2022,burgholzerSimulationPathsQuantum2022,burgholzerEfficientConstructionFunctional2021`, _compilation_ {cite:p}`zulehnerEfficientMethodologyMapping2019,hillmichExploitingQuantumTeleportation2021,zulehnerCompilingSUQuantum2019,willeMappingQuantumCircuits2019,burgholzerLimitingSearchSpace2022,pehamDepthoptimalSynthesisClifford2023, schneiderSATEncodingOptimal2023,pehamOptimalSubarchitecturesQuantum2023,quetschlichMQTPredictorAutomatic2023,quetschlichCompilerOptimizationQuantum2023,quetschlichReducingCompilationTime2023,schoenbergerUsingBooleanSatisfiability2024,schoenbergerCyclebasedShuttlingTrappedIon2024,schoenbergerShuttlingScalableTrappedIon2024,schmidHybridCircuitMapping2024,schmidComputationalCapabilitiesCompiler2024,matoAdaptiveCompilationMultilevel2022,matoCompilationEntanglingGates2023,matoCompressionQubitCircuits2023,matoMixeddimensionalQuditState2024`, and _verification_ {cite:p}`burgholzerAdvancedEquivalenceChecking2021,burgholzerImprovedDDbasedEquivalence2020, burgholzerPowerSimulationEquivalence2020,burgholzerRandomStimuliGeneration2021, burgholzerVerifyingResultsIBM2020,pehamEquivalenceCheckingParameterized2023,pehamEquivalenceCheckingQuantum2022,pehamEquivalenceCheckingParadigms2022,willeVerificationQuantumCircuits2022` of quantum circuits.
+Reaching towards the hardware level, we also consider tools for _quantum error correction_ {cite:p}`berentDecodingQuantumColor2023,berentSoftwareToolsDecoding2023,strikisQuantumLowdensityParitycheck2023` and _physical design_ {cite:p}`kunasaikaranFrameworkDesignRealization2024`.
```

Comment:
for the QEC work also the bosonic qldpc paper might be relevant is self-advertisement is ok here :P 

@article{berent2023analog,
  title={Analog information decoding of bosonic quantum LDPC codes},
  author={Berent, Lucas and Hillmann, Timo and Eisert, Jens and Wille, Robert and Roffe, Joschka},
  journal={arXiv preprint arXiv:2311.01328},
  year={2023}
}